### DISCUSSION TITLE
Zwift Hub Compatability

### DISCUSSION THREAD

**Discussion by ikim67** [cagnulein/qdomyos-zwift#1089] [Ideas]
Hi Roberto,

 I tested the application last night - Beta Version 2.12.15. Didn't have a chance to play with the features but it does indeed connect to the Zwift Hub now. Great stuff. Hopefully will get some more time this week to dig in and have a real play! Thank you!

 One thing I did think of. We often have to index the bikes when taking on and off the trainers. Its easier to do it off the bike because the default resistance makes it hard to turn the peddles in the small cog but nicer to do on the trainer. Perhaps, and maybe its there cos I haven't played yet, a setting called 'Bike Indexing' would be good. It would just lower the resistance of the direct trainer to make indexing easier when switched on.

![image](https://user-images.githubusercontent.com/48229504/208683200-54ec3af3-6bd1-4b63-8b7f-b5c8272ed5b4.png)

**Comment by cagnulein** [comment]
Great @ikim67 !

> a setting called 'Bike Indexing' would be good. It would just lower the resistance of the direct trainer to make indexing easier when switched on.

you can use the minus button on the resistance tile for this already, isn't it?

**Comment by ikim67** [comment]
I'll give that a try! I only had about 30 mins last night and wanted to
knock out one of my plan rides. Wanted to test that the app connected first
though.

Will have a little more time later in the week to play with it some more
and test with things like the GTA5 Bike mod too! :-)

**Comment by Benzinbruder** [comment]
Hi @cagnulein!

Either I'm blind, stupid or there just is no way to pm you on the Zwift forums.
So here I am trying my luck here. Could you please provide me with a code for the newest beta? I'd love to try your app with the Zwift Hub!

**Reply by cagnulein** [reply]
hi @Benzinbruder 
send me an email to roberto.viola83 at gmail.com indicating this ticket and if you are on android or iOS. Thanks
